src/PharoThings-Devices-HCSR04.package/PotHCSR04Device.class/instance/readDistance.st

```diff
@@ -0,0 +1,4 @@
+commands
+readDistance
+	self sendPulse.
+	^ (self readPinStateDuration * 34300 / 2 / 1000000) asFloat round: 1
```

Comment:
Would be nice to have a comment for those numbers.
And for readPinStateDuration to understand how sensor is working (why time to get value 1 in pin is a kind of distance).

Reply:
I put some comments in the last PR.